api-report/purchases-js.api.json

```diff
@@ -1,7 +1,7 @@
 {
   "metadata": {
     "toolPackage": "@microsoft/api-extractor",
-    "toolVersion": "7.50.1",

```

Comment:
@vicfergar can you bump the version of the api-extractor tool on your machine and run again the generation?

Reply:
Since I removed the new `ErrorCodes`, there are no changes in the API report

Reply:
BTW I noticed that we're pinging an older version in our `packages.json`, that's why it gets changed in this case

> "@microsoft/api-extractor": "^7.48.0",